fem/gridfunc.cpp

```diff
@@ -998,6 +1004,8 @@ void GridFunction::GetVectorValue(ElementTransformation &T,
          FaceElementTransformations * FET =
             dynamic_cast<FaceElementTransformations *>(&T);
 
+         MFEM_ASSERT(FET != nullptr,
```

Comment:
Either add a newline after this for consistency with the prior two, or remove them all? I'ldd be in favor of no newline after the `FET =  ...`, newline after `MFEM_ASSERT` for all 3.